scipy/special/tests/test_basic.py

```diff
@@ -716,22 +716,46 @@ def test_nbdtrin(self):
     def test_ncfdtr(self):
         assert_equal(cephes.ncfdtr(1,1,1,0),0.0)
 
+    @pytest.mark.xfail(
+        reason=(
+            "ncfdtr uses a Boost math implementation but ncfdtri"
+            "inverts the less accurate cdflib implementation of ncfdtr."
+        )
+    )
     def test_ncfdtri(self):
         assert_equal(cephes.ncfdtri(1, 1, 1, 0), 0.0)
         f = [0.5, 1, 1.5]
         p = cephes.ncfdtr(2, 3, 1.5, f)
```

Comment:
Can we make it a little more obvious that this is not actually using Cephes by (importing and) using ufuncs by the usual name? 
```suggestion
        p = scu.ncfdtr(2, 3, 1.5, f)
```

Reply:
Yeah, but I think it should be in a separate PR for all of them. This is a historical remnant from when Travis first did special and it only wrapped cephes. I guess no one wanted to change all of the cephes to special or something, so this has been like this for 20+ years.

Reply:
I don't mean for all occurences in the file, just this one, since the comment specifically says "ncfdtr uses a Boost math implementation" whereas it looks like it's using `cephes`.

Reply:
I know, I mean I’d rather fix it for all of them at once in a different PR. This didn’t come from cephes before this PR either. I think most of these things don’t come from cephes. Someone just decided to make cephes an alias for special to avoid having to do a find and replace. What we have now is at least consistent with the mad logic of this file. I can follow up with a PR fixing it for everything after this one goes in if you’d be willing to review.

Reply:
At my computer now, (but hopefully you're asleep). The reason I'd rather keep this as `cephes` until their all fixed instead of making just `ncfdtr` say `special`  is because then in this block it will make it look doubly so that all of the functions except `ncfdtr` listed here come from `cephes`, when out of the seven of them, only `nbdtr` does.

```python
    def test_nbdtr(self):
        assert_equal(cephes.nbdtr(1,1,1),1.0)

    def test_nbdtrc(self):
        assert_equal(cephes.nbdtrc(1,1,1),0.0)

    def test_nbdtri(self):
        assert_equal(cephes.nbdtri(1,1,1),1.0)

    def test_nbdtrik(self):
        cephes.nbdtrik(1,.4,.5)

    def test_nbdtrin(self):
        assert_equal(cephes.nbdtrin(1,0,0),5.0)

    def test_ncfdtr(self):
        assert_equal(special.ncfdtr(1,1,1,0),0.0)

    @pytest.mark.xfail(
        reason=(
            "ncfdtr uses a Boost math implementation but ncfdtri"
            "inverts the less accurate cdflib implementation of ncfdtr."
        )
    )
    def test_ncfdtri(self):
        assert_equal(cephes.ncfdtri(1, 1, 1, 0), 0.0)
        f = [0.5, 1, 1.5]
        p = cephes.ncfdtr(2, 3, 1.5, f)
        assert_allclose(cephes.ncfdtri(2, 3, 1.5, p), f)
```

Reply:
K. 
I'd be willing to review the PR if it's just fine and replace.